tests/plugins/conditional_formatting.test.ts

```diff
@@ -935,6 +935,108 @@ describe("conditional formats types", () => {
       });
     });
 
+    test("Operator GreaterThan with simple reference", () => {
+      model.dispatch("ADD_CONDITIONAL_FORMAT", {
+        cf: {
+          rule: {
+            type: "CellIsRule",
+            operator: "GreaterThan",
+            values: ["=A2"],
+            style: { fillColor: "#ff0f0f" },
+          },
+          id: "11",
+        },
+        ranges: toRangesData(sheetId, "A1:B2"),
+        sheetId,
+      });
+      setCellContent(model, "A1", "1");
+      setCellContent(model, "B1", "2");
+      setCellContent(model, "A2", "0");
+      setCellContent(model, "B2", "4");
+      expect(getStyle(model, "A1")).toEqual({ fillColor: "#ff0f0f" });
+      expect(getStyle(model, "B1")).toEqual({});
+      expect(getStyle(model, "B1")).toEqual({});
+      expect(getStyle(model, "B2")).toEqual({ fillColor: "#ff0f0f" });
+    });
+
+    test("Operator GreaterThan with fixed simple reference", () => {
+      model.dispatch("ADD_CONDITIONAL_FORMAT", {
+        cf: {
+          rule: {
+            type: "CellIsRule",
+            operator: "GreaterThan",
+            values: ["=$A$2"],
```

Comment:
I would go as far as testing the different combinations of fixed col/ row  (a$2, $A2)